iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m

```diff
@@ -98,9 +90,12 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 #define ONESIGNAL_APP_ID_KEY_FOR_TESTING @"ONESIGNAL_APP_ID_KEY_FOR_TESTING"
 
 + (NSString*)getOneSignalAppId {
+    NSString* newAppId = @"0ba9731b-33bd-43f4-8b59-61172e27447d";
     NSString* onesignalAppId = [[NSUserDefaults standardUserDefaults] objectForKey:ONESIGNAL_APP_ID_KEY_FOR_TESTING];
-    if (!onesignalAppId)
-        onesignalAppId = @"0ba9731b-33bd-43f4-8b59-61172e27447d";
+    if (![newAppId isEqualToString:onesignalAppId]) {
```

Comment:
Is ok to always check to newAppId? shouldn't we use the saved one if it exists?

Reply:
if the new one matches the existing one it will use the new one. This helps with easily switching between two and caching the new one being used within the SDK